hack/generate-manifest-windows.sh

```diff
@@ -62,6 +64,10 @@ case $key in
     RUNTIME="containerd"
     shift
     ;;
+    --ovs-containerd)
+    RUNTIME="ovs-containerd"
```

Comment:
ditto, I think this will be confusing too. ovs is included in runtime info, while it actually is about whether to run OVS as Pod. I think a parameter like "--install-ovs-pod", "--ovs-run-mode <pod|service>".

Reply:
changed to --ovs-run-mode